README.md

```diff
@@ -1,65 +1,118 @@
-### How to build: (run from root dir)
+<img height="90" alt="MongoDB Atlas JDBC Driver" align="right" src="resources/media/MongoDBAtlasJDBC.png" />
+
+# MongoDB Atlas JDBC Driver
```

Comment:
```suggestion
# MongoDB Atlas SQL JDBC Driver
```

Reply:
Done. I also updated the next occurrences I found in the file.